query/functions/usage.feature

```diff
@@ -0,0 +1,471 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#noinspection CucumberUndefinedStep
+Feature: Function call positions behaviour
```

Comment:
Great - though it's probably not just this. Perhaps we have two files? `definition.feature`, and then you can roll in stuff to do with calling them with invalid nr of arguments etc into `basic.feature`?

Reply:
Acutally the function call stuff can go into the one you've called `signature`, which is also includes mostly function call validation.

Reply:
Definition yes.
Basic is basic function execution.